selector/methods/partition.py

```diff
@@ -443,6 +443,18 @@ def __init__(
             Index for the sample to start selection from; this index is the first sample selected.
         scaling: float
             Percent of average maximum distance to use when eliminating the closest points.
+
+        Notes
+        -----
+        The `Mediod` implementation is based on the KDTree algorithm and therefore can give
+        different results for cases with duplicated points or the same features for differenr
```

Comment:
Fix typo. I didn't have push access to your forked repo, so I couldn't change it.

Reply:
Problem resolved.